### DISCUSSION TITLE
If no user  , go to '/auth'  ,  if user exists go to  '/'

### DISCUSSION THREAD

**Discussion by thanhtutzaw** [gladly-team/next-firebase-auth#567] [Q&A]
How can I achieve this . 
if no user  then go to '/auth'  ,  if user exists go to  '/'

I struggling withAuthUser , authAction
Any example ?

**Comment by kmjennison** [comment]
See the example app here:
https://github.com/gladly-team/next-firebase-auth/tree/v1.x/example

And a live demo:
https://nfa-example-git-v1x-gladly-team.vercel.app/

**Reply by thanhtutzaw** [reply]
how to access useAuthUser()   in getServerSideProps

**Reply by kmjennison** [reply]
Again, please look at the example:
https://github.com/gladly-team/next-firebase-auth/blob/v1.x/example/pages/ssr-auth-required.js#L47-L52
